debian/aarch64/jessie/entry.sh

```diff
@@ -65,16 +71,20 @@ function init_non_systemd()
 	# trap the stop signal then send SIGTERM to user processes
 	trap signal_handler SIGRTMIN+3 SIGTERM
 
-	udevd & 
-	udevadm trigger &> /dev/null
-	
 	CMD=$(which "$1")
 	# echo error message, when executable file doesn't exist.
 	if [ $? == '0' ]; then
 		shift
-		"$CMD" "$@" &
-		pid=$!
-		wait $pid
+		if [ ! -z "$RESIN" ] && [ ! -z "$RESIN_DEVICE_UUID" ]; then
+			# run this on resin device only
+			start_udev
+			
+			"$CMD" "$@" &
+			pid=$!
+			wait $pid
+		else
+			exec "$CMD" "$@"
```

Comment:
Unfortunately we can't use `exec` without breaking `docker stop`. Since we declared that we need signal RTMIN+3 the user process won't know how to handle it. So we need to use `wait $pid` on this case too. Only udev should be kept in the conditional